generators/config/templates/default.yml

```diff
@@ -21,13 +21,17 @@
  maximized_window: true # Specify maximized browser window size
 
  # Specify one of the following drivers: selenium, selenium_grid, webkit, poltergeist, phantomjs, sauce,
- # testingbot, browserstack
+ # testingbot, browserstack, headless_chrome
  driver: phantomjs
 
  # -Selenium-
  # specify one of next browsers: iexplore (ie), firefox (ff), chrome, safari
  selenium_browser: ff
 
+ # -Headless Chrome-
+ # List of available flags (http://peter.sh/experiments/chromium-command-line-switches/)
+ flags: [--window-size=1920x1080, --disable-gpu]
```

Comment:
We try to keep the settings simple due to we can pass them via environment variable. So, we have 2 options for refactoring:

1) put back as hard coded flags to config/capybara.rb
2) or change it like that:

```
headless_chrome_flags: "--window-size=1920x1080, --disable-gpu"
```
Update ruby code in config accordingly (injecting the setting to array in code)